README.md

```diff
@@ -6,9 +6,35 @@ Codidact is community-driven Q&A software: created, maintained, used, and run by
 money-making company behind this.
 
 We're currently in the early stages of development, but once we have a working MVP, we'll be hosting a public Codidact instance
-for everyone to use. The software will also be available to download and run yourself to host your own instance under your own
+for everyone to use. The software will also be available to download and run yourself to host your instance under your own
 rules.
 
+## Installation
+These instructions are for setting up and running a local instance of Codidact.
+
+#### Windows
+We'll be using Visual Studio for this setup.
+
+1. [Download Visual Studio 2019 Community](https://visualstudio.microsoft.com/downloads/) and start up the installer. If you
+   already have VS2019 installed, run the Visual Studio Installer instead.
+2. When prompted to select workloads, select the "ASP.NET and Web Development" workload, and then install it.
+3. Once the installation is complete, clone/download this repository and extract it somewhere.
+4. In the root of the repository, open the `Codidact.sln` file with visual studio to get started.
+5. Once Visual Studio has finished loading, you can run Codidact by setting the WebUI project as the startup project.
+
+Alternatively, if you don't want to run Visual Studio just to start the project, you can navigate to the src/WebUI folder
+(so that `WebUI.csproj` is in your working directory) and issue the command `dotnet run`.
+
+#### Linux
+1. [Download .NET Core SDK](https://docs.microsoft.com/en-us/dotnet/core/install/linux-package-manager-ubuntu-1904). Specifically,
+   from this page, follow the sections labeled "Register Microsoft Key and feed" and "Install the .NET Core SDK"
+2. Once the installation is complete, clone/download the repository and extract it somewhere.
+3. You can run Codidact by navigating to the src/WebUI folder (so that `WebUI.csproj` is in your working directory) and issuing
+   the command `dotnet run`.
+
+#### Technology Stack
+A list of the current tech stack is [here](https://github.com/codidact/docs/wiki/Technology-Stack).
```

Comment:
I'd add something along the lines of "You'll need to install these versions in your environment, before you can start following the above process". It might also be a good idea to move it to the top.

Reply:
Maybe the wording should be "We'll install these items in the steps below, but if all of these are already installed on your system you can skip straight to running the project"? That way the list is still available if somebody doesn't want to / can't follow the provided steps, but we have an "EZ setup" guide (for lack of a better term)